app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -256,11 +263,11 @@ interface ProjectViewModel {
         private val rewardsToolbarTitle = BehaviorSubject.create<Int>()
         private val scrimIsVisible = BehaviorSubject.create<Boolean>()
         private val setInitialRewardPosition = BehaviorSubject.create<Void>()
-        private val showBackingFragment = BehaviorSubject.create<Project>()
+        private val showBackingFragment = PublishSubject.create<Project>()
```

Comment:
let's use a `BehaviorSubject` and see if it affects rotation

Reply:
So this will affect rotation, here's the scenario:

ok yes i remember why I changed them to `BehaviorSubject`!
so let's say you view an unbacked project
`showRewardsFragment` will emit
then you back it
`showBackingFragment` will emit
then you cancel it
`showRewardsFragment` will emit again

now you rotate and the project is refreshed
if they're `BehaviorSubject`s both will re-emit but that's not what we want
we only want one to emit